native/src/modules/app/StateType.js

```diff
@@ -57,20 +57,14 @@ export type PoiRouteConfigType = {|
   +city: string
 |}
 
-// eslint-disable-next-line flowtype/type-id-match
-type LanguageKey = string
-// eslint-disable-next-line flowtype/type-id-match
-type Path = ?string // Path can be falsy if the current displayed view is a list of events or POIs
-type AllAvailableLanguagesType = $ReadOnlyMap<LanguageKey, Path>
```

Comment:
I think it makes the code more readable to keep the string typedefs. Path should become PathType ofc